### ISSUE TITLE
Figure out how to differentiate through long-wave radiation

### ISSUE THREAD

**Issue by shoyer** [climate-analytics-lab/jax-gcm#23]
Speedy uses a look-up table to calculate long-wave radiation from temperature, e.g.,
https://github.com/samhatfield/speedy.f90/blob/0ebfdbfcd62efd46dd1802284908645be4b230be/source/longwave_radiation.f90#L83

We can implement this in JAX, but look-up tables are discontinuous, so gradients won't be well-defined. We might consider using an alternative that defines gradients like `jax.numpy.interp` instead.
